src/generator/operations.ts

```diff
@@ -284,17 +285,47 @@ function generateOperation(clientName: string, op: Operation, imports: ImportMan
     text += `\tif err != nil {\n`;
     text += `\t\treturn nil, err\n`;
     text += `\t}\n`;
+    if (!op.responses) {
+      text += '\tresult := &HttpResponse{\n';
```

Comment:
This should be `HTTPResponse` else the linter will complain.

Reply:
Fixed